client/src/screens/group.screen.js

```diff
@@ -0,0 +1,210 @@
+    paddingVertical: 4,
+  },
+  header: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+});
+
+// format createdAt with moment
+const formatCreatedAt = createdAt => moment(createdAt).calendar(null, {
+  sameDay: '[Today]',
+  nextDay: '[Tomorrow]',
+  nextWeek: 'dddd',
+  lastDay: '[Yesterday]',
+  lastWeek: 'dddd',
+  sameElse: 'DD/MM/YYYY',
+});
+
+const Header = ({ onPress, onPressFind }) => (
+  <View style={styles.header}>
+  </View>
+);
+Header.propTypes = {
+};
+
+class Member extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { id, username } = this.props.member;
+    console.log(this.props)
+    return (
+      <TouchableHighlight
+        key={id}
+      >
+        <View style={styles.groupContainer}>
+          <Icon name="user" size={24} color={'blue'} />
+          <View style={styles.groupTextContainer}>
+            <View style={styles.groupTitleContainer}>
+              <Text style={styles.groupName} numberOfLines={1}>{username}</Text>
+              <Text style={styles.groupLastUpdated}>{id}</Text>
+            </View>
+            <Text style={styles.groupUsername}>
+            </Text>
+            <Text style={styles.groupText} numberOfLines={1}>
+            </Text>
+          </View>
+        </View>
+      </TouchableHighlight>
+    );
+  }
+}
+
+Member.propTypes = {
+  member: PropTypes.shape({
+    id: PropTypes.number.isRequired,
+    username: PropTypes.string.isRequired,
+  }),
+};
+
+class Group extends Component {
+  static navigationOptions = {
+    title: 'Group',
+    tabBarIcon: ({ tintColor}) => <Icon size={24} name={'group'} color={tintColor} />
+  };
+
+  constructor(props) {
+    super(props);
+    this.onRefresh = this.onRefresh.bind(this);
+  }
+
+  onRefresh() {
+    this.props.refetch();
+  }
+
+  keyExtractor = item => item.id;
+
+  renderItem = ({ item }) => <Member member={item}/>;
+
+
+  render() {
+    const { loading, user, networkStatus } = this.props;
+    // render loading placeholder while we fetch messages
+    if (loading || !user) {
+      return (
+        <View style={[styles.loading, styles.container]}>
+          <ActivityIndicator />
+        </View>
+      );
+    }
+
+    // render group stuff
+    return (
+      <View style={styles.container}>
+      <Text style={styles.warning}>{this.props.navigation.state.params.title}</Text>
+      <Text style={styles.warning}>{this.props.navigation.state.params.tags}</Text>
+
+      <FlatList
+        //TODO: Make a query that pulls just the one group back, not all of them
+        //find the array object with the same name as the one passed
+        data={user.organisation.groups.find((element) => {if(element.name == this.props.navigation.state.params.title){return element}}).users}
+        keyExtractor={this.keyExtractor}
+        renderItem={this.renderItem}
+        ListHeaderComponent={() => <Header onPress={this.goToNewGroup} onPressFind={this.goToSearchGroup} />}
```

Comment:
no capital letter for parameters: `listHeaderComponent` not `ListHeaderComponent`

Reply:
I just used what was in the tutorial? weird that it would be wrong and still work.